kubernetes-model/kubernetes-model/src/main/java/io/fabric8/kubernetes/api/model/Quantity.java

```diff
@@ -46,86 +48,201 @@
 @Buildable(editableEnabled = false, validationEnabled = false, generateBuilderPackage=true, builderPackage = "io.fabric8.kubernetes.api.builder", inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done"))
 public class Quantity  implements Serializable {
 
-    private String amount;
-    private String format;
-    private Map<String, Object> additionalProperties = new HashMap<String, Object>();
+  private String amount;
+  private String format;
+  private Map<String, Object> additionalProperties = new HashMap<String, Object>();
 
-    /**
-     * No args constructor for use in serialization
-     *
-     */
-    public Quantity() {
-    }
+  /**
+   * No args constructor for use in serialization
+   *
+   */
+  public Quantity() {
+  }
 
-    /**
-     * Single argument constructor for setting amount.
-     *
-     * @param amount amount of quantity specified.
-     */
-    public Quantity(String amount) {
-        this.amount = amount;
-    }
+  /**
+   * Single argument constructor for setting amount.
+   *
+   * @param amount amount of quantity specified.
+   */
+  public Quantity(String amount) {
+    AbstractMap.SimpleEntry<String, String> amountFormatPair = splitAmountAndFormat(amount);
+    this.amount = amountFormatPair.getKey();
+    this.format = amountFormatPair.getValue();
+  }
 
-    /**
-     * Double argument constructor for setting amount along with format.
-     *
-     * @param amount amount of quantity specified
-     * @param format format for the amount.
-     */
-    public Quantity(String amount, String format) {
-        this.amount = amount;
-        this.format = format;
+  /**
+   * Double argument constructor for setting amount along with format.
+   *
+   * @param amount amount of quantity specified
+   * @param format format for the amount.
+   */
+  public Quantity(String amount, String format) {
+    this.amount = amount;
+    this.format = format;
+  }
+
+  public String getAmount() {
+    return amount;
+  }
+
+  public void setAmount(String amount) {
+    this.amount = amount;
+  }
+
+  public String getFormat() {
+    return format;
+  }
+
+  public void setFormat(String format) {
+    this.format = format;
+  }
+
+  public static BigDecimal getNormalizedValue(Quantity quantity) {
```

Comment:
Is this returning the quantity in bytes? why not just call it `getAsBytes` or  `getAmountInBytes`?